contracts/examples/digital-cash/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+#![no_std]
+#![allow(unused_attributes)]
+
+use elrond_wasm::{imports, require, sc_error};
+imports!();
+
+mod deposit_info;
+use deposit_info::DepositInfo;
+
+pub const SECONDS_PER_ROUND: u64 = 6; 
+
+#[elrond_wasm_derive::contract(DepositImpl)]
+pub trait Deposit {
+	
+	#[init]
+	fn init(&self) {
+		let my_address: Address = self.get_caller();
+		self.set_owner(&my_address);
+	}
+
+	fn get_expiration_round(&self,valability: u64) -> u64{
+		let valability_rounds = valability / SECONDS_PER_ROUND;
+		
+		return self.get_block_round() + valability_rounds;
```

Comment:
Use `self.get_block_timestamp()` for keeping track of the expiration instead of `self.get_block_round()`. That means you'll work with timestamps everywhere and not need conversions between rounds / seconds.

Reply:
this problem was discussed and it was decided that block round it is the best solution